src/main/resources/db/changelog/create/0014_EIP1-6984_modify_rejection_notes_structure.xml

```diff
@@ -0,0 +1,131 @@
+
+    <changeSet author="tomos.slater@softwire.com" id="0014_EIP1-6984_modify_postal_rejection_notes_structure" context="ddl">
+        <addColumn tableName="postal_vote_application_english_rejection_reasons">
+            <column name="elector_reason" type="varchar(100)" afterColumn="rejection_reasons"/>
+            <column name="include_in_comms" type="boolean" afterColumn="elector_reason"/>
+        </addColumn>
+
+        <sql>
+            UPDATE postal_vote_application_english_rejection_reasons
+            SET postal_vote_application_english_rejection_reasons.elector_reason = postal_vote_application_english_rejection_reasons.rejection_reasons,
+                postal_vote_application_english_rejection_reasons.include_in_comms =
+                    CASE WHEN postal_vote_application_english_rejection_reasons.elector_reason IN ('FRAUDULENT_APPLICATION', 'OTHER_REJECT_REASON') THEN FALSE ELSE TRUE END,
+                postal_vote_application_english_rejection_reasons.rejection_reasons =
+                    CASE postal_vote_application_english_rejection_reasons.rejection_reasons
+                        WHEN 'The applicant is not registered to vote' THEN 'NOT_REGISTERED_TO_VOTE'
+                        WHEN "The applicant's identity has not been confirmed" THEN 'IDENTITY_NOT_CONFIRMED'
+                        WHEN 'The applicant is not eligible for reserved polls' THEN 'NOT_ELIGIBLE_FOR_RESERVED_POLLS'
+                        WHEN 'A date of birth was not provided' THEN 'DOB_NOT_PROVIDED'
+                        WHEN 'The application is incomplete' THEN 'INCOMPLETE_APPLICATION'
+                        WHEN 'The signature was not acceptable' THEN 'SIGNATURE_IS_NOT_ACCEPTABLE'
+                        WHEN 'Suspected or possible fraudulent applicant' THEN 'FRAUDULENT_APPLICATION'
+                        ELSE 'OTHER_REJECT_REASON' END
+        </sql>
+
+        <addColumn tableName="postal_vote_application_welsh_rejection_reasons">
+            <column name="elector_reason" type="varchar(100)" afterColumn="rejection_reasons"/>
+            <column name="include_in_comms" type="boolean" afterColumn="elector_reason"/>
+        </addColumn>
+
+        <sql>
+            UPDATE postal_vote_application_welsh_rejection_reasons
+            SET postal_vote_application_welsh_rejection_reasons.elector_reason = postal_vote_application_welsh_rejection_reasons.rejection_reasons,
+                postal_vote_application_welsh_rejection_reasons.include_in_comms =
+                    CASE WHEN postal_vote_application_welsh_rejection_reasons.elector_reason IN ('FRAUDULENT_APPLICATION', 'OTHER_REJECT_REASON') THEN FALSE ELSE TRUE END,
+                postal_vote_application_welsh_rejection_reasons.rejection_reasons =
+                    CASE postal_vote_application_welsh_rejection_reasons.rejection_reasons
+                        WHEN 'The applicant is not registered to vote' THEN 'NOT_REGISTERED_TO_VOTE'

```

Comment:
@StephenPenney I looked through the postal/proxy and this API for anywhere using the welsh messages here and couldn't find any, so I don't know what to map from. Do you know where I could find them?

Reply:
I don't think we are currently populating the welsh ones and it's fine to assume any Welsh ones would be in English as lacking the translation